<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QMainWindow" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SUPR गाउ खाने कथा</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/supr_logo.png</normaloff>:/supr_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1251</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">*{
font-family : century gothic;
font-size: 24px;
background-color: rgb(0, 136, 255);
}

#loginPage{
	border-image: url(:/password.jpg);

}
#signupPage
{

	border-image: url(:/password.jpg);
}



QFrame
{
background-color: rgb(255, 255, 255);
border-radius : 10px;
}

QPushButton 
{
	background-color: rgb(255, 110, 31);
border-radius : 8px;
color : white;
}

QPushButton:hover
{
background-color: rgb(41, 206, 16);
color :white;
}



QLabel {
color : black;
font: 13pt &quot;Segoe UI&quot;;
}

QLineEdit 
{
background : transparent;
border : 1px solid black;
border-radius : 4px;
}

#signupFrame , #loginFrame
{
border: 2px solid rgb(44, 44, 44);
}

#welcome{

	font: 700 30pt &quot;Segoe UI&quot;;
}

#supr{
background:transparent;
font:700 40pt;
	color: rgb(255, 255, 255);
}

#startBtn{
	background-color: #FF6E1F;
color:white;
font: 700 10pt ;
}

#alertMsg 
{
font-size:10pt;
background-color:transparent;
border:none;
color:red;
}

#alertMsg2
{
font-size:10pt;
background-color:transparent;
border:none;
color:red;
}

#startBtn:hover
{
background-color: rgb(41, 206, 16);
color :white;
}

#gauKhane
{
	background-color: rgb(255, 35, 50);
}

#gauKhane:hover
{
background-color: rgb(41, 206, 16);
color :white;
}

#underCon
{
	background-image: url(:/welcomeBack.png);
}

#questionOne
{
	border-image: url(:/gau_khane.png);
}


#question
{
background:transparent;
font:700 30pt;
	color: rgb(255, 255, 255);
}

QRadioButton
{
background-color:transparent;
	font: 700 17pt &quot;Segoe UI&quot;;
color:white;
}


#nextBtn
{
	background-color: rgb(29, 63, 255);
}

#nextBtn:hover
{
background-color: rgb(41, 206, 16);
color :white;
}

#backBtn2
{
	background-color: rgb(255, 47, 50);
}

#backBtn2:hover
{
background-color: rgb(41, 206, 16);
color :white;
}

#backBtn
{
	background-color: rgb(255, 47, 50);
}

#backBtn:hover
{
background-color: rgb(41, 206, 16);
color :white;
}








</string>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QFrame" name="loginFrame">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>491</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 15pt ;
color : black;</string>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginBtn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>300</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt ;</string>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="loginEmail">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="loginPw">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>220</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>190</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>350</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Forgot Password?</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>410</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 17px &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Need an account?</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signupLink">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>410</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
text-decoration: underline;
font: 12pt;
color : black;
font: 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 700 9.5pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Database Status...</string>
       </property>
      </widget>
      <widget class="QLabel" name="alertMsg2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>460</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="signupPage">
     <widget class="QFrame" name="signupFrame">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>30</y>
        <width>491</width>
        <height>571</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 15pt ;
color : black;</string>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
      </widget>
      <widget class="QPushButton" name="signupBtn">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>410</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 700 12pt ;</string>
       </property>
       <property name="text">
        <string>SIGN UP</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="signupEmail">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>210</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>180</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="signupPw">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>320</y>
         <width>331</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>290</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>470</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 17px &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Already have account?</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginLink">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>470</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
text-decoration: underline;
font: 12pt;
color : black;
font: 14pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>520</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;
font: 700 9.5pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Database Status...</string>
       </property>
      </widget>
      <widget class="QLabel" name="alertMsg">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>370</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>First Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="firstName">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lastName">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>110</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>80</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property>
       <property name="text">
        <string>Last Name</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="welcomePage">
     <property name="styleSheet">
      <string notr="true">#welcomePage
{
	
	border-image: url(:/welcomeBack.png);
}</string>
     </property>
     <widget class="QLabel" name="supr">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>260</y>
        <width>831</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&amp;quot; SUPR गाउ खाने कथा &amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="welcome">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>190</y>
        <width>381</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;WELCOME TO&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="startBtn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>480</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt ;</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="catPage">
     <property name="styleSheet">
      <string notr="true">#catPage
{
	border-image: url(:/welcomeBack.png);
}
</string>
     </property>
     <widget class="QLabel" name="choose">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>90</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#choose
{
	font: 700 20pt &quot;Segoe UI&quot;;
}</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;CHOOSE&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginBtn_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>250</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt ;</string>
      </property>
      <property name="text">
       <string>SCIENCE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginBtn_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>310</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt ;</string>
      </property>
      <property name="text">
       <string>SPORTS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginBtn_5">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>430</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt ;</string>
      </property>
      <property name="text">
       <string>ESPORTS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loginBtn_6">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>490</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt ;</string>
      </property>
      <property name="text">
       <string>HISTORY</string>
      </property>
     </widget>
     <widget class="QPushButton" name="gauKhane">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>370</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt ;</string>
      </property>
      <property name="text">
       <string>गाउ खाने कथा</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="underCon">
     <widget class="QLabel" name="under">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>290</y>
        <width>831</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:transparent;
font:700 40pt;
	color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;UNDER CONSTRUCTION...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>440</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt ;</string>
      </property>
      <property name="text">
       <string>BACK</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="questionOne">
     <widget class="QLabel" name="question">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>210</y>
        <width>831</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;टाउकोले लेख्ने, पुच्छरले मेटने, के हो ?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>350</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>इस्केल</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>350</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>इरेजर</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>430</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>शार्प्नर</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>430</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>पेन्सिल</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextBtn">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>540</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt ;</string>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtn2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>540</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 12pt ;</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
